src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
+        <sky-demo-page-component-state stateName="Context menu" imagePath="~/assets/img/guidelines/dropdown/option-type-context.png"></sky-demo-page-component-state>
+        <sky-demo-page-component-state stateName="Icon" imagePath="~/assets/img/guidelines/dropdown/option-type-icon.png"></sky-demo-page-component-state>
+      </sky-demo-page-component-states>
+        <h3>Button style</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>Dropdowns can use the default or primary style inherited from SKY UX buttons. See the <a routerLink="/design/guidelines/page-layouts" fragment="primary-action-guidelines">primary action guidelines</a> for details about when to use the primary style.</p>
+        </div>
+    </sky-demo-page-content>
+      
+    <sky-demo-page-content sectionHeading="Behavior and states">
+        <h3>Behavior</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>The behavior of the dropdown menu is defined by the <a routerLink="/components/popover">popover</a> component.</p>
+        </div>
+        <h3>States</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>Dropdowns inherit the styling for their different states (hover, disabled, etc.) from <a routerLink="/components/button">SKY UX buttons</a>.</p>
+        </div>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Content">
+      <h3>Dropdown menu action text</h3>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>If an action clearly applies to the object associated with the context menu, use the following format for the action wording:</p>
+          <p><span class="sky-emphasized">&lt;Verb&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/context-menu.png" alt="context menu actions: View, Edit, Delete" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">&lt;Verb&gt;</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/dropdown-single-action.png" alt="dropdown menu Edit: Addresses, Email addresses, Phone numbers" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with more than one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">More</span></p>
```

Comment:
I think we should reverse the order here and list the button icon first and then the button text. That matches the left-to-right/top-to-bottom way that users will visually take in the image. I'll update that, but I can undo the change if anyone feels strongly about it.